### ISSUE TITLE
[Component] [autocomplete] Autocomplete on android not updating value

### ISSUE THREAD

**Issue by smilosevic** [element-plus/element-plus#10002]
Bug Type: **`Component`**

## Environment

- Vue Version: `3.2.40`
- Element Plus Version: `2.2.17`
- Browser / OS: `Chrome 106.0.5249.79`
- Build Tool: `Vite`

### Related Component

- `el-autocomplete`

### Reproduction Link

[Docs](https://element-plus.org/en-US/component/autocomplete.html)

### Steps to reproduce

Start typing in autocomplete

### What is Expected?

Suggestions to get filtered

### What is actually happening?

Nothing

### Additional comments

Search is not updating on android devices. I tried Pixel 6 pro, samsung 21, lg v60. Both on chrome and mozilla.
Search is updated only after typing space character, closing keyboard or removing all characters.
Works on ios.

Link to video recording:
https://photos.google.com/share/AF1QipMH385BRp_-EXrUEJyPkujR-CaFMATidFy4mqSRtJ3y3rXizpl3DqNXPdQMJGvq4Q/photo/AF1QipMcmXjK6nTXN80xseornBaqz0q9z9H-0s_7zbKS?key=c1ZscktsM0NUWFZoSTkwY2ZiV241ZXpTc29PXzhB

**Comment by chenxch** [comment]
I think this may be a problem with the input method. On some PCs, the input method also has a similar problem because the corresponding keyboard event cannot be triggered.

**Comment by saslavik** [comment]
This is because inside the autocomplete there is an input that has a methods:
-         @compositionstart="handleCompositionStart"
-         @compositionupdate="handleCompositionUpdate"
-         @compositionend="handleCompositionEnd"
And 
```
      handleCompositionStart() {
        this.isComposing = true;
      },
      handleCompositionUpdate(event) {
        const text = event.target.value;
        const lastCharacter = text[text.length - 1] || '';
        this.isComposing = !isKorean(lastCharacter);
      },
      handleCompositionEnd(event) {
        if (this.isComposing) {
          this.isComposing = false;
          this.handleInput(event);
        }
      },
```

My solution:
```
<el-autocomplete
    ref="autocompleteRef"
    v-model="value">
    
    <setup>
    const autocompleteRef = ref()
    const isComposingVal = computed(() => autocompleteRef.value?.$refs?.input?.isComposing)
    watch(() => isComposingVal.value,
      () => {
        if (autocompleteRef.value?.$refs?.input?.isComposing) {
          autocompleteRef.value.$refs.input.isComposing = false
        }
      })
     return {value, autocompleteRef}
</setup>
```

**Comment by kuka-radovan** [comment]
I am facing this problem too. Is there any update to this issue?

@saslavik sadly your solution does not work due to missing `isComposing` among exposed fields.
https://github.com/element-plus/element-plus/blob/dev/packages/components/input/src/input.vue#L495

**Comment by kuka-radovan** [comment]
I would kindly ask for an update on this issue... are there any plans to work on this at all?

**Comment by saslavik** [comment]
@kuka-radovan yep, because this solution for [element.io](https://element.eleme.io/#/en-US) for Vue 2 (not element-plus)

**Comment by saslavik** [comment]
@kuka-radovan You need to look on CompositionUpdate. Maybe need timeout, because compositionUpdate with nextTick still have old value. With setTimeout around 5ms we have new data. and if you have a component with <el-input v-bind="$attrs"> and your props has modelValue:
`const compositionupdate = (event) => {
      setTimeout(() => {
        const value = event.target.value
        nextTick(() => {
          emit('update:modelValue', value)
        })
      }, 10)
    }`

I don't know if this is working, but need to try :)
